src/Pelagos/Bundle/AppBundle/Resources/public/js/dataset-submission.js

```diff
@@ -727,6 +727,18 @@ $(function() {
                 source.append(option);
                 source.append(sortOptions(source.find("option").detach()));
             }
+        } else if ($(event.currentTarget).text() == "up") {
+            var selectedOption = target.find("option:selected");
+            var prevOption = selectedOption.prev("option");
+            if (prevOption.is("option")) {
+                selectedOption.detach().insertBefore(prevOption);
+            }
+        } else if ($(event.currentTarget).text() == "down") {
```

Comment:
Use '===' :)

Reply:
This was just copied from the lines above. Maybe we have to change it everywhere?
I'm hesitant to change it for a language like Javascript using jQuery.  $.text() always returns a string, is it really necessary to check to type too.  Triple equals are good if you can expect something else, and want to make sure.

Reply:
I guess === is better.